README.md

```diff
@@ -1 +1,24 @@
-# wikibase-rest-fantasies
\ No newline at end of file
+# wikibase-rest-fantasies
+
+Prototype of the schema of the future Wikibase REST API.
+
+## Scope
+
+The REST API is meant to provide the functionality currently offer by Wikibase "Action API".
+The scope of the schema is limited to Wikibase core Action API functionality, i.e. Wikibase's additions to MediaWiki's own API actions are intentionally not covered.
+
+### Work in progress
+
+The following elements are not yet defined:
+* Response schema
+* Complete definition of filtering parameters
+* ....
+
+### Not covered
+
+The following elements are considered non essential for the design decisions and are intentionally excluded from the prototype schema:
+* TBD
+
+## Preview
+
+REST API schema can be previewed using [Swagger Editor](https://editor.swagger.io/). Simply paste in the contents of the `openapi.json` file.
```

Comment:
This isn't how it works unfortunately, the spec is "compiled" from all of these files and the online editor doesn't know how to resolve that. that's why we created the publish script, so that you could just go to petstore.swagger.io and then paste the url to the already resolved full spec:  `https://raw.githubusercontent.com/wmde/wikibase-rest-fantasies/gh-pages/openapi.json`

Reply:
Ideally we would host our own instance of swagger UI so that people could simply browse the spec, but that was out of scope for the prototype week.

Reply:
thanks, I knew it cannot be that easy. Updated the instructions.

> Ideally we would host our own instance of swagger UI so that people could simply browse the spec, but that was out of scope for the prototype week.

I think it is fine to do it later, and document the existing options for the time being.

Reply:
Got a little annoyed by having to give people convoluted instructions so I played with the petstore; deeplinks are possible like so:
https://petstore.swagger.io/?url=https://raw.githubusercontent.com/wmde/wikibase-rest-fantasies/gh-pages/openapi.json

Reply:
neat. I've ended up changing the URL https://wikibase-rest-fantasies.wmflabs.org/ which @Silvan-WMDE provided.